### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Log Query Output_Color Contrast: In Dark Theme, Color contrast ratio for red color content in the console window is 3.2:1 which is < 4.5:1

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3568]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #MAS1.4.3; #ColorContrast; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application in Dark theme
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Open Eclipse and install the Scala plug-in 
4) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5) Tab till File-->New-->Project and press enter
6) In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight Local Run Sample (Scala)'. Tab till Next and press enter
7) Tab till Next button and press enter and press Finish button
8) In Project explorer tab, right click on  src-->logquery.scala and tab till run as-->1 scala application. 
9)Verify the Color contrast ratio for red color content in the console window is >= 4.5:1 or not

**Actual Result:** 
In Dark Theme, Color contrast ratio for red color content in the console window is 3.2:1 which is < 4.5:1

**Expected Result:** 
In Dark Theme, Color contrast ratio for red color content in the console window should be >= 4.5:1

**User Impact:** 
Low vision users will find difficult to read the content if the text does not meet the standard color contrast ratio

**MAS Reference:** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_15_Color Contrast for the red color content is 4 51](https://user-images.githubusercontent.com/41109135/65323918-48a18d80-dbc8-11e9-8e5a-cc342bc83592.png)

**Comment by t-rufang** [comment]
The color of the error message is controlled by Eclipse. When we run a project in Eclipse, and there are some errors, the error message will show as below. As you can see, there are also color contrast issue.
![image](https://user-images.githubusercontent.com/32627233/68447351-96c32c80-021a-11ea-89f2-93da12b169d9.png)
![image](https://user-images.githubusercontent.com/32627233/68447356-9cb90d80-021a-11ea-847b-57bbb27ead1c.png)

Therefore, this issue should be owned by Eclipse and it's out of the scope of our toolkit.

**Comment by t-rufang** [comment]
- Importance of experience: Medium,  Users with low vision will not be able to see the error message in the console window properly.
- Quality of experience: not impacted. Eclipse provide alter theme for users, for instance, light, which has more clear fonts for error messages.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)
- [External link for the bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=550841)

**Comment by Tanushagottipati** [comment]
Closing this bug as approved by SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749679
